future_challenges_for_RSEng.tex

```diff
@@ -573,7 +572,6 @@ \subsection{Transformations in Academic Institutions}
 some RSEs could focus on code modernisation, clean-up, and modularisation,
 which could pay off in faster and easier development in the future.
 This is an opportunity that would not be possible if relying solely on short-term funded, isolated RSEs.
-We hope that a change of culture in research about the acceptance of research software has happened.
```

Comment:
Maybe something like:
We expect that research has fully recognized the importance of research software.

Reply:
i think here, we do `hope` that that is the case